manifests/plugin/ansible.pp

```diff
@@ -15,8 +15,16 @@
     ],
   }
 
+  if ! $pulpcore::apache_https_vhost {
+    fail('HTTPS must be turned on for Ansible content')
+  } elsif $pulpcore::apache::https_port != 443 {
+    $external_content_url = "https://${pulpcore::servername}:${pulpcore::apache::https_port}${pulpcore::apache::content_path}"
+  } else {
+    $external_content_url = "https://${pulpcore::servername}${pulpcore::apache::content_path}"
+  }
+
   pulpcore::plugin { 'ansible':
-    config        => "ANSIBLE_API_HOSTNAME = \"${pulpcore::servername}\"\nANSIBLE_CONTENT_HOSTNAME = \"${pulpcore::servername}\"",
+    config        => "ANSIBLE_API_HOSTNAME = \"${pulpcore::servername}\"\nANSIBLE_CONTENT_HOSTNAME = \"${external_content_url}/\"",
```

Comment:
Just wondering: does it need a trailing slash?

Reply:
I assumed it was needed. However, I just tested this without a trailing slash. It appears that pulp3 handles this without the trailing slash without any issues.